src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
+    @Override
+    public List<Object> get() {
```

Comment:
The `get()` method should return a List of `Truck` objects, not a List of `Object`. Correct it to `public List<Truck> get()`.